src/test/java/uk/ac/ebi/ena/webin/xml/conversion/EnaSraConverterTest.java

```diff
@@ -2536,8 +2536,7 @@ public void testAnalysisXml_ReferenceAlignmentAnalysisType_ConversionToJson()
             + "    \"checksum\" : \"9840f585055afc37de353706fd31a377\",\n"
             + "    \"fileName\" : \"Bmi1_bw.bam\",\n"
             + "    \"checksumMethod\" : \"MD5\"\n"
-            + "  } ],\n"
-            + "  \"attributes\" : [ ]"
+            + "  } ]\n"
```

Comment:
i dont like this either but the converted/generated JSONs sometimes do this for some reason. if you were able to figure out how to stop generating empty objects and arrays in produced JSONs and the unit tests continue to pass them great job.

can i ask why leave sequencingProject: {} instead of completely removing it?